### ISSUE TITLE
Cannot access French audio

### ISSUE THREAD

**Issue by tmortagne** [sarbes/plugin.video.artemediathek#3]
Kodi version: 19
Plugin version: 2.0.1 installed from official "Kodi Add-on repository"

My system and Kodi are in French and while the list does show the French version of the programs I always end up with German audio (and no possibility to change the audio stream) when I go for example to Most Recent and choose the first program ("La conquête" right now).

**Comment by ltai0001** [comment]
@tmortagne any feedback on this? Ran into the same issue trying to get French audio. Arte has an app though the Apple Store and in there I can switch the audio between different languages so hoping this addon can be fixed

**Comment by etieng** [comment]
Hi, this add-on sounds really great !
I'm facing the same issue, no way to switch audio to French, and also, only german subtitles are available.
Any config option in a file or something ?

**Comment by tmortagne** [comment]
> @tmortagne any feedback on this?

Not really. I did not really dig much. I could not find any working solution for Arte on Kodi recently, so I'm using the official website these days (but the imagine quality seems lower).
